apps/pie-docs/src/components/textarea/overview/overview.md

```diff
@@ -0,0 +1,447 @@
+    {% contentPageImage {
+      src: "../../../assets/img/components/textarea/filled.svg",
+      width: "256px",
+      alt: "Textarea component with a label and string content inside it."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+### Height
+
+The height, unlike the width, of a textarea is flexible, with two different options available.
+
+{% contentLayout %}
+  {% contentItem %}
+    <h4>Auto-resize</h4>
+    <p>Used when the intended behaviour is for the form's height to dynamically adjust based on input length.</p>
+    {% contentPageImage {
+      src: "../../../assets/img/components/textarea/default.svg",
+      width: "256px",
+      alt: "Textarea component with auto-resizing option."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h4>Manual-resize</h4>
+    <p>Used when the intended behavior is for the user to have control over the form’s height.</p>
+    {% contentPageImage {
+      src: "../../../assets/img/components/textarea/manual-resize.svg",
+      width: "256px",
+      alt: "Textarea component with manual-resizing option."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+---
+
+## Sizes
+
+Use a consistent height when it is being used alongside other form components on the same page.
+
+{% contentLayout %}
+  {% contentItem %}
+    <h4>Small</h4>
+    <p>Use when there isn’t enough vertical space for the default textarea size.</p>
+    {% contentPageImage {
+      src: "../../../assets/img/components/textarea/small.svg",
+      width: "206px",
+      alt: "Textarea component with a small height."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h4>Medium</h4>
+    <p>Default size and should be used whenever possible.</p>
+    {% contentPageImage {
+      src: "../../../assets/img/components/textarea/medium.svg",
+      width: "206px",
+      alt: "Textarea component with a medium height."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h4>Large</h4>
+    <p>Choose this size when there is a lot of space to work with. This size is typically used in simple forms or when a textarea is placed by itself on a page.</p>
+    {% contentPageImage {
+      src: "../../../assets/img/components/textarea/large.svg",
+      width: "206px",
+      alt: "Textarea component with a large height."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+#### Auto-resize
+
+{% componentDetailsTable {
+  tableData: autoResize
+} %}
+
+#### Manual-resize
+
+{% componentDetailsTable {
+  tableData: manualResize
+} %}
+
+---
+
+## Content
+
+### Placeholder
+
+Use clear placeholder text for the text input trigger so that users understand the purpose.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/textarea/placeholder.svg",
+    alt: "Textarea component with a placeholder.",
+    width: "256px"
+} %}
+
+### Input types
+
+#### Alphanumeric
+
+Both letters and numbers are allowed, for all use-cases this will be used.
```

Comment:
For the text input we also mentioned special characters. Omitting this makes it sound to me like they are not allowed.

Reply:
```suggestion
Both letters, numbers and certain special characters are allowed, for the majority of use-cases this will be used.
```